hw/ip/otbn/dv/otbnsim/sim/load_elf.py

```diff
@@ -3,7 +3,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 '''OTBN ELF file handling'''
-
```

Comment:
Not a big deal but since you're going to have to push again I'd suggest dropping this hunk.

Reply:
Do you mean I've to add a blank line after line 5?